docs/software/apps-and-envs/gaussian.md

```diff
@@ -0,0 +1,141 @@
+# Gaussian
+
```

Comment:
[Gaussian](https://gaussian.com/gaussian16/) is a general-purpose computational chemistry software package with focus on electronic structure modeling. Gaussian is developed and licensed by Gaussian, Inc.